chart/kubeapps/README.md

```diff
@@ -210,8 +223,69 @@ kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceac
 helm init --service-account tiller
 ```
 
-It is also possible, though less common, that your cluster does not have Role Based Access Control (RBAC) enabled. If this is the case you should perform the chart installation by setting `rbac.create=false`:
+It is also possible, though less common, that your cluster does not have Role Based Access Control (RBAC) enabled. To check if your cluster has RBAC you can execute:
+
+```console
+$ kubectl api-versions
+```
+
+If the above command does not include entries for `rbac.authorization.k8s.io` you should perform the chart installation by setting `rbac.create=false`:
 
 ```console
 $ helm install --name kubeapps --namespace kubeapps bitnami/kubeapps --set rbac.create=false
 ```
+
+### Upgrade process fail
+
+It is possible that when upgrading Kubeapps an error appears. Something like:
+
+```console
+Error: configmaps "kubeapps-internal-apprepository-jobs-bootstrap" already exists
```

Comment:
I'm not sure if we should put this in the current chart readme now that the bootstrap job is no longer there. We have the migration guide, so I think it might be enough. What do you think? cc @migmartri

Reply:
Yeah, let's remove the specific error message